Mathport/Syntax/Translate/Notation.lean

```diff
@@ -75,7 +75,7 @@ def NotationDesc.toKind (n4 : Name) : NotationDesc → NotationKind
 open NotationKind in set_option hygiene false in
 def predefinedNotations : HashMap String NotationEntry := [
     ("exprProp", const <| Id.run `(Prop)),
-    ("expr $ ", binary fun f x => Id.run `($f $ $x)),
+    ("expr $ ", binary fun f x => Id.run `($f <| $x)),
```

Comment:
What's up with this? Is this the new standard now? I prefer `$`, and it's confusing that they both exist.

Reply:
I can revert this line, but `<|` is certainly the standard choice in core Lean 4 now.  Presumably because `$x` now has a new meaning.  I agree that it is confusing to have both `$` and `<|` with exactly the same expansion and the same precedence.

Reply:
Let's take this to zulip to get some consensus.